src/site/content/en/notifications/use-push-notifications-to-engage-users/index.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Use push notifications to engage and re-engage users
+authors:
+  - katejeffreys
+description: |
+  Use push notifications to engage users with targeted, timely updates.
+date: 2019-10-21
+codelabs:
+  - codelab-notifications-get-started
+  - codelab-notifications-service-worker
+---
+
+## Why use push notifications?
+
+Notifications present small chunks of information to a user. Web apps can use notifications to tell users about important, time-sensitive events, or actions the user needs to take. 
+
+The look-and-feel of notifications varies between platforms. For example: 
+
+<figure class="w-figure">
+  <img src="./predicaments-android.png" alt="A notification on an android device.">
+  <figcaption class="w-figcaption">A notification on an android device.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./predicaments-macbook.png" alt="A notification on a MacBook.">
+  <figcaption class="w-figcaption">A notification on a MacBook.</figcaption>
+</figure>
+
+Traditionally, web browsers had to initiate the exchange of information between server and client by making a request. [Web push technology](https://developer.mozilla.org/en-US/docs/Web/API/Push_API), on the other hand, lets you configure your server to send notifications when it makes sense for your app. A push service creates unique URLs for each subscribed service worker. Sending messages to a service worker's URL raises events on that service worker, prompting it to display a notification.      
```

Comment:
Rather than reference "unique URLs" I'd probably use "unique push subscriptions".

This would then change "Sending messages to a service worker's URL" to "Sending messages to a users push subscriptions".

Reply:
+1